docs/mm_quick_start.md

```diff
@@ -0,0 +1,181 @@
+will be located.
+This ensures the Triton SDK container has access to the model
+config variants that Model Analyzer creates.<br><br>
+**Important:** You must ensure the absolutes paths are identical on both sides of the mounts<br><br>
+
+## `Step 3:` Profile both models concurrently
+
+---
+
+The [examples/quick-start](../examples/quick-start) directory is an example
+[Triton Model Repository](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_repository.md) that contains two libtorch models: `add_sub` & `resnet50_libtorch`
+
+Run the Model Analyzer `profile` subcommand inside the container with:
+
+```
+model-analyzer profile \
+    --model-repository <path-to-examples-quick-start> \
+    --profile-models add_sub,resnet50_libtorch --triton-launch-mode=docker \
+    --output-model-repository-path <path-to-output-model-repo>/<output_dir>
+    --export-path profile_results
+    --run-config-profile-models-concurrently-enable
+```
+
+**Important:** You must specify an `<output_dir>` subdirectory. You cannot have `--output-model-repository-path` point directly to `<path-to-output-model-repo>`
+
+**Important:** If you already ran this earlier in the container, you can use the `--override-output-model-repository` option to overwrite the earlier results.
+
+**Important**: All models must be in the same directory
+
+**Important:** The checkpoint directory should be removed between consecutive runs of
+the `model-analyzer profile` command.<br><br>
+
+---
+
+The `--run-config-profile-models-concurrently-enable` option tells MA to load and optimize both models concurrently using the [Quick Search](config_search.md#quick-search-mode) algortihm.
+
+This will profile both models concurrently using the quick search algorithm to find the maximal throughput gain for both models by iterating across instance counts and batch sizes. By default, the algorithm is attempting to find the best balance of gains for each model, not the best combined total throughput.
+
+After the quick search completes, MA will then sweep concurrencies for the top three configurations and then create a summary report and CSV outputs.
+
+---
+
+Here is an example result summary, run on a TITAN RTX GPU:
+
+![Result Summary Top](../examples/mm_result_summary_top.jpg)
+![Result Summary Table](../examples/mm_result_summary_table.jpg)
+
+You will note that the top model configuration has a lower total combined throughput than either the 2nd or 3rd configurations. However, it does a better job at increasing the total percentage throughput gain versus the other configurations.
+
+For example, comparing the top config (42,24) to the third best (37,19), you will see that while the gain in `add_sub` throughput is large (~1600 infer/sec) it is only a 3.5% increase. However, this is offset by a decrease in `resnet50_libtorch` throughput (~75 infer/sec), which represents an 8% decrease in throughput. Therefore, the (42,24) is deemed to be a better configuration by MA.
```

Comment:
It wasn't clear to me what 42,24 and 37,19 were. I thought they might be throughput values or batch sizes. I see now they are config names. You might want to clarify.

Reply:
Good point. I've changed this to be the full config name.